Source/Constant.swift

```diff
@@ -166,3 +166,7 @@ internal enum Constant {
         }
     }
 }
+
+extension Constant {
```

Comment:
I wouldn't follow this pattern in open source projects. This should move to the location where it. Also, because they're in different files I have no idea what this value is unless I navigate to this file.

Reply:
Yeah, makes sense, I'll move it

Reply:
Thank you!